pyaerocom/web/test/test_helpers.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jun  8 10:13:05 2020
+
+@author: jonasg
+"""
+from pyaerocom.web import helpers as h
+import pytest
+from contextlib import contextmanager
+
+
+@contextmanager
+def does_not_raise_exception():
+    yield
+
+@pytest.mark.parametrize('input_args,expectation', [
+    pytest.param(dict(), pytest.raises(ValueError)),
+    # the following input for obs_vars is converted into a
+    # list in the setup class
+    pytest.param(dict(obs_id='Bla',
+                      obs_vars='od550aer',
+                      obs_vert_type='Column'), does_not_raise_exception()),
+    pytest.param(dict(obs_id='Bla',
+                      obs_vars='od550aer',
+                      obs_vert_type='Slant-Column'), pytest.raises(ValueError)),
+
+    ])
+def test_ObsConfigEval(input_args,expectation):
+    with expectation:
+        cfg = h.ObsConfigEval(**input_args)
+        for key, val in input_args.items():
+            if key=='obs_vars' and isinstance(val, str):
+                val = [val]
+            assert cfg[key] == val
```

Comment:
@ejgal in #137 you may also use an approach like this to parametrize test cases where some are failing and some are not.